packages/ui5-proxy-middleware/src/base/utils.ts

```diff
@@ -195,36 +195,40 @@ export const setHtmlResponse = (res: any, html: string): void => {
  * @param args list of runtime arguments
  * @returns The content of the manifest.json
  */
-export const getManifest = async (args: string[]): Promise<Manifest> => {
+export async function getManifest(args: string[]): Promise<Manifest | undefined> {
     const projectRoot = process.cwd();
     const yamlFileName = getYamlFile(args);
     const ui5YamlPath = join(projectRoot, yamlFileName);
     const webAppFolder = await getWebAppFolderFromYaml(ui5YamlPath);
     const manifestPath = join(projectRoot, webAppFolder, 'manifest.json');
-    const manifest: Manifest = JSON.parse(readFileSync(manifestPath, { encoding: 'utf8' }));
-
-    return manifest;
-};
+    if (existsSync(manifestPath)) {
+        return JSON.parse(readFileSync(manifestPath, { encoding: 'utf8' }));
+    } else {
+        return undefined;
+    }
+}
```

Comment:
Could be deleted, see other comment

Reply:
https://github.com/SAP/open-ux-tools/pull/1054/commits/e66599a445bf869d90bb045cd8e762f64fbb6128